x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curations_history/components/ignored_queries_panel/ignored_queries_panel.tsx

```diff
@@ -84,7 +84,12 @@ export const IgnoredQueriesPanel: React.FC = () => {
             )}
           </EuiFlexItem>
           <EuiFlexItem component="span" grow={false}>
-            <EuiButtonEmpty iconType="refresh" size="xs" onClick={() => loadIgnoredQueries()}>
+            <EuiButtonEmpty
+              data-test-subj="enterpriseSearchIgnoredQueriesPanelRefreshButton"
```

Comment:
@efegurkan author of the `event_generating_elements_should_be_instrumented` ESLint rule here. I noticed this particular rule is activated for `x-pack/plugins/enterprise_search` in https://github.com/elastic/kibana/blob/main/.eslintrc.js#L1527. If you have a different format it might make sense to remove this while any changes are made. I'm happy to collaborate btw on any changes.

Reply:
I reverted the linter changes. they were auto fixed by mistake while trying to auto fix the errors, it auto fixed the warnigns. please take another look :pray: